src/js/index.js

```diff
@@ -0,0 +1,72 @@
+const menuListWrapper = document.querySelector("#espresso-menu-list");
+const menuInput = document.querySelector("#espresso-menu-name");
+const confirmBtn = document.querySelector("#espresso-menu-submit-button");
+const countElem = document.querySelector(".menu-count");
+
+const data = {
+  menuList: [],
+};
+
+const createBtnElem = (btnText, btnType) => {
+  const btn = document.createElement("button");
+
+  btn.textContent = btnText;
+  btn.classList = `bg-gray-50 text-gray-500 text-sm mr-1 menu-${btnType}-button`;
+  return btn;
+};
+
+const createMenuItemElem = (menuName) => {
+  const li = document.createElement("li");
+  const span = document.createElement("span");
+  const soldOutBtn = createBtnElem("품절", "sold-out");
+  const editBtn = createBtnElem("수정", "edit");
+  const removeBtn = createBtnElem("삭제", "remove");
+
+  span.textContent = menuName;
+  span.className = "w-100 pl-2 menu-name";
+
+  editBtn.addEventListener("click", () => {
+    const editMenuIdx = li.getAttribute("data-menu-id");
+    const menuName = prompt("메뉴명을 수정하세요 :)");
+
+    data.menuList.splice(editMenuIdx, 1, createMenuItemElem(menuName));
+    rendorMenuList();
```

Comment:
```suggestion
    const menuName = prompt("메뉴명을 수정하세요 :)");

    if (menuName) {
        data.menuList.splice(editMenuIdx, 1, createMenuItemElem(menuName));
        rendorMenuList();
    }
```

`prompt`를 취소한다면 데이터에 `null` 값이 들어갈 수도 있지 않을까요 ?

Reply:
생성 시에는 빈 값 - 또는 빈 문자열 - 이 검증되어야하지만, 수정에서는 필수요구사항은 아니었군요! 👍